webapp/test/clj/lipas/backend/handler_test.clj

```diff
@@ -572,6 +572,32 @@
     (is (= 200 (:status resp1)))
     (is (= 200 (:status resp2)))))
 
+(deftest search-order-test
+  (let [sites    (for [site-name ["\"Bantis\" beachvolleykenttä 2"
```

Comment:
[minor] You could write both test-data generation and seeding within a single `doseq`. E.g.

```clojure
(doseq [site-name "..." "..."]
  (let [site  (-> (tu/gen-sports-site)
                    (assoc :name site-name))]
    (core/index! search site :sync)))
```

I personally prefer using a `truthy` and describing keywords instead of boolean flags, therefore `:sync` instead of `true`. :) Otherwise you need to interpret the meaning of the flag by reading the source / docs.

Reply:
Moved things to single doseq, much cleaner!